app/validators/partner_details/answers_validator.rb

```diff
@@ -1,47 +1,40 @@
 module PartnerDetails
   class AnswersValidator < BaseAnswerValidator
-    def validate
-      return unless applicable?
-      return if complete?
-
-      # NOTE: partner_details refers to general partnership information, stored
-      # in partner and partner_detail
-      errors.add :partner_details, :incomplete
-      errors.add :base, :incomplete_records
-    end
+    include TypeOfMeansAssessment
 
     def applicable?
       crime_application.client_has_partner == 'yes'
     end
 
-    # rubocop:disable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity
     def complete?
       return false unless record
-      return true if no_partner_complete?
-      return false if no_partner_incomplete?
 
-      [
-        record.relationship_to_partner,
-        crime_application.partner&.first_name,
-        crime_application.partner&.last_name,
-        crime_application.partner&.date_of_birth,
-        record.involvement_in_case,
-        nino?,
-        conflict_of_interest?,
-        same_address?,
-        address?,
-      ].map(&:present?).all?(true)
+      validate
+      errors.empty?
     end
-    # rubocop:enable Metrics/MethodLength, Metrics/AbcSize, Metrics/CyclomaticComplexity
 
-    private
+    def validate # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity, Metrics/PerceivedComplexity
```

Comment:
Refactored this file to match how we validate the other sections so we can potentially display specific errors in the future